packages/theming/apple-theme/src/createAppleTheme.ios.ts

```diff
@@ -1,15 +1,12 @@
 import { ThemeReference } from '@fluentui-react-native/theme';
 import { Theme } from '@fluentui-react-native/theme-types';
 import { Appearance } from 'react-native';
-import { BaseAppleDarkThemeIOS, BaseAppleLightThemeIOS } from './appleTheme.ios';
+import { getBaseAppleThemeIOS } from './appleTheme.ios';
 
 export function createAppleTheme(): ThemeReference {
-  const baseAppleTheme = () => {
-    const current = Appearance.getColorScheme();
-    return current === 'light' ? BaseAppleLightThemeIOS : BaseAppleDarkThemeIOS;
-  };
-
-  const appleThemeReference = new ThemeReference({} as Theme, baseAppleTheme);
+  const appleThemeReference = new ThemeReference({} as Theme, () => {
+    return getBaseAppleThemeIOS(Appearance.getColorScheme() === 'light');
+  });
```

Comment:
Could we keep the behavior above of having isLight be its own variable? Helps with future debugging :)